spotbugs-exclude-filter.xml

```diff
@@ -0,0 +1,12 @@
+<FindBugsFilter>
+    <Match>
+        <!-- Class is not instantiable -->
+        <Class name="com.hubspot.guice.transactional.impl.TransactionalConnection"/>
+        <Bug pattern="CT_CONSTRUCTOR_THROW"/>
+    </Match>
+    <Match>
+        <!-- New noisy checks introduced in spotbugs -->
+        <Bug pattern="EI_EXPOSE_REP,EI_EXPOSE_REP2"/>
```

Comment:
Should we add these filters to [basepom-policy](https://github.com/HubSpot/basepom-policy/blob/master/src/main/resources/spotbugs/spotbugs-suppress.xml), rather than needing to add exclusions to each repo individually?

Reply:
👍 And for any other isolated suppressions, it might be less code to just do `@SuppressWarnings("...")` on the relevant code block, assuming spotbugs will respect that